src/views/registration-js/button-checkout.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * This template renders the attendee registration checkout button
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration/button-checkout.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ */
+if ( ! $checkout_url ) {
+	return;
+}
+?>
+<form
+	class="tribe-block__tickets__registration__checkout"
+	action="<?php echo esc_url( $checkout_url ); ?>"
+	method="post"
+>
+	<input type="hidden" name="tribe_tickets_checkout" value="1" />
+	<button
+		type="submit"
+		class="button-primary tribe-block__tickets__registration__checkout__submit"
+		<?php if ( $cart_has_required_meta && ! $is_meta_up_to_date ) : ?>
+		disabled
```

Comment:
Should this use `tribe_disabled()`? Sorry not familiar enough with the new function or what it does specifically, but saw you used disabled attribute manually here.

Reply:
Disabled by default, so no.

Reply:
hm, actually, we could - probably should. even though it's by default. Note this illustrates the main reason I made the function - I left off the `aria-disabled`.